package/scripts/postinstall

```diff
@@ -29,9 +29,9 @@ export HOME="${homebrew_directory}"
 # reset Git repository
 cd "${homebrew_directory}"
 git config --global --add safe.directory "${homebrew_directory}"
-git reset --hard
 git checkout --force master
-git branch | grep -v '\*' | xargs -n 1 git branch --delete --force || true
+git reset --hard "$(git describe --tags "$(git rev-list --tags --max-count=1)")"
```

Comment:
`git checkout --force master` here leaves the Git repo at the `master` branch. I think we probably want to checkout a `stable` branch instead (with `git checkout --force -B stable`)? This is also what we do [in the shell script installer](https://github.com/Homebrew/install/blob/8a20a36edbb80a7a80f5eb2038672df9a911e3ad/install.sh#L971).

Reply:
Makes sense. And if the installation has developer mode enabled, it'll switch back to `master` on the next `brew update`. 
I also changed the tag-finding command to match the installer's.